app/models/validations/property_validations.rb

```diff
@@ -41,6 +41,8 @@ def validate_uprn(record)
 
   def validate_property_postcode(record)
     postcode = record.postcode_full
+    return unless postcode
```

Comment:
When we try to clear the postcode as part of the "clear all" for compound validations, we hit this validation.

The validation will still trigger fine when nothing is entered from the single log submission flow, cause postcode would come through as an empty string instead of a nil